torch/ao/quantization/quantize_pt2e.py

```diff
@@ -66,6 +69,22 @@ def prepare_qat_pt2e(
     model = _disallow_eval_train(model)
     return model
 
+_QUANT_OPS = [
+    torch.ops.quantized_decomposed.quantize_per_tensor.default,
+    torch.ops.quantized_decomposed.quantize_per_tensor.tensor,
```

Comment:
this should include quantize_per_channel as well

Reply:
oh yeah I updated this locally let me update the PR